english/yellow-baby-buggy-bumpers/yellow-baby-buggy-bumpers.md

```diff
@@ -10,6 +10,9 @@ your career but you know it's the right thing to do.
 
 You then start walking towards the amusement park.
 
-As you approach the park, you get a foreboding sense that not all is right. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.